### PR TITLE
Bring pre-6.0 blocks & components back

### PR DESCRIPTION
**Pull request by imath**
It's more convenient to maintain all blocks from this repository.

- To do so we added some hooks to use all BuddyPress blocks from this plugin instead of the BuddyPress ones.
- We are also using a namespace so that it will be easier to merge into BP Core blocks/functions when they will be ready to.
- Use Sass to generate the member & group blocks css.
- Apply improvements Renato added to plugin organization.

### CODE REVIEW

File: src/bp-members/js/blocks/member/edit.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	}
+
+	return (
+		<Fragment>
+			<BlockControls>
+				<Toolbar>
+					<ToolbarButton
+						icon="edit"
+						title={ __( 'Select another member', 'buddypress' ) }
+						onClick={ () =>{
```
Comment (renatonascalves) [inline src/bp-members/js/blocks/member/edit.js:84]:
```suggestion
						onClick={ () => {
```

File: src/bp-groups/js/blocks/group/edit.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+	}
+
+	return (
+		<Fragment>
+			<BlockControls>
+				<Toolbar>
+					<ToolbarButton
+						icon="edit"
+						title={ __( 'Select another group', 'buddypress' ) }
+						onClick={ () =>{
```
Comment (renatonascalves) [inline src/bp-groups/js/blocks/group/edit.js:85]:
```suggestion
						onClick={ () => {
```

**Review by renatonascalves — APPROVED** [review]
Two suggestions! But otherwise, looks good!

**Comment by imath** [comment]
Thanks a lot for your review @renatonascalves 👍

**Pull request merged**